[ci] verible-format check broken

It looks like the recent changes to the util/verible-format.sh script broke the CI check that uses this script:

```
ci/scripts/verible-format.sh
========================== Starting Command Output ===========================
/bin/bash --noprofile --norc /home/vsts/work/_temp/46ac84be-d201-4572-a848-35686cd8d6f9.sh
usage: verible-verilog-format [options] <file> [<file...>]
To pipe from stdin, use '-' as <file>.
##[error]Verilog format with Verible failed. Run 'util/verible-format.sh' to check and fix all errors.
This flow is currently experimental and failures can be ignored.
##[error]Bash exited with code '1'.
Finishing: Check formatting of files on allow list with Verible
```

CC @msfschaffner (If you have time, otherwise I'll look tomorrow.)

--- a/ci/scripts/verible-format.sh
+++ b/ci/scripts/verible-format.sh
@@ -7,7 +7,7 @@
 
 set -e
 
-util/verible-format.sh || {
+util/verible-format.sh --allowlist || {
   echo -n "##vso[task.logissue type=error]"
   echo "Verilog format with Verible failed. Run 'util/verible-format.sh' to check and fix all errors."
   echo "This flow is currently experimental and failures can be ignored."

